### PR TITLE
Refactoring translate to only show a manual translate

### PR DESCRIPTION
**Pull request by Bri-B**

### CODE REVIEW

File: src/components/Customer/Profile/Translate.jsx [hunk 1]
```diff
@@ -1,14 +1,16 @@
+/* eslint-disable no-restricted-syntax */
```
Comment (jstrzesz) [inline src/components/Customer/Profile/Translate.jsx]: why is this here? we shouldn't be using eslint-disable

**Review by lschwall — APPROVED** [review]

**Review by tenhold — APPROVED** [review]

**Review by christophermbooth — APPROVED** [review]
